.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Get version and SHA
+        id: version-sha
+        run: |
+          echo "##[set-output name=version;]$(cat VERSION)"
+          echo "##[set-output name=sha_id;]$(git rev-parse --short HEAD)"
+      - name: Docker login
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Docker build, tag, and push
+        run: |
+          echo "Building version ${IMAGE_VERSION} of image ${IMAGE_NAME}"
+          docker build -t ${IMAGE_NAME} .
+          docker tag "${IMAGE_NAME}" "${IMAGE_NAME}:${IMAGE_VERSION}"
```

Comment:
looks like we need a loop for the `:VERSION` suffix

Reply:
What do you mean exactly?

Reply:
we copy paste the same lines just changing the version of the image to tag/push

Reply:
Okay I'll improve it when I convert them to buildx